libcst/metadata/scope_provider.py

```diff
@@ -667,6 +657,7 @@ def __init__(
         self.name = name
         self.node = node
         self._scope_overwrites = {}
+        self._name_prefix = self._make_name_prefix()
```

Comment:
Pyre is unhappy with this line. Any suggessions @zsol

Reply:
Seems like a bug to me, might be worth raising with Pyre folks internally